src/main/resources/swagger/api-docs.yaml

```diff
@@ -202,6 +202,37 @@ paths:
         500:
           description: Internal Error
   /workspaces/{workspaceNamespace}/{workspaceName}/submissions/{submissionId}:
+    delete:
+      tags:
+      - Submissions
+      operationId: deleteSubmission
+      summary: |
+        delete a submission
+      produces:
+      - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: submissionId
+          description: Submission ID
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
```

Comment:
We don't use Rawls as a model for what we design as endpoint urls in orchestration. This is a different scenario as we are returning whatever Rawls does. If Rawls says the response is a 204, then our api docs should also say that.